src/utils/conf.rs

```diff
@@ -58,7 +51,8 @@ impl Conf {
         }
         s.add_source(
             Environment::with_prefix("hyle")
-                .separator("_")
+                .separator("__")
+                .prefix_separator("_")
```

Comment:
it looks like a breaking change on the usage of env var, could we have a small ping on this when merging ?

Reply:
This should actually be a breaking fix, which env vars are you using?

Reply:
in the tools/docker-compose.yml

Reply:
A priori `HYLE_CONSENSUS_SLOT_DURATION` c'est sensé pas marcher, dans mes tests les variables avec des `_` marche pas car la config l'interprète comme un séparateur.

Après la PR ça devrait être `HYLE_CONSENSUS__SLOT_DURATION`.

Mais du coup je modifie la PR

Reply:
Pour le reste ça change rien, c'est juste le séparateur de nesting qui est impacté

Reply:
étrange car c'est moi qui ait ajouté cet env var et à priori ça faisait bien ce que je voulais :thinking: